app/src/main/java/com/example/awesomefish/ui/stages/GameHostActivity.kt

```diff
@@ -97,7 +103,7 @@ class GameHostActivity : AppCompatActivity(), PauseMenu.PauseMenuItemClickedList
     }
 
     private fun showQuitDialog() {
-        AlertDialog.Builder(this)
+        /*AlertDialog.Builder(this)
             .setTitle(R.string.quit_dialog_title)
```

Comment:
Remove the comment. This block is not useful anymore. Let's delete the commented code.